cmd/containerd-shim-runhcs-v1/service.go

```diff
@@ -13,18 +13,20 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func beginActivity(activity string, fields logrus.Fields) {
-	logrus.WithFields(fields).Info(activity)
+func beginActivity(activity string, fields logrus.Fields) *logrus.Entry {
+	log := logrus.WithFields(fields)
+	log.Info(activity)
+	return log
 }
 
-func endActivity(activity string, fields logrus.Fields, err error) {
+func endActivity(log *logrus.Entry, activity string, err error) {
 	if err != nil {
-		fields["result"] = "Error"
-		fields[logrus.ErrorKey] = err
-		logrus.WithFields(fields).Error(activity)
+		log.Data["result"] = "Error"
```

Comment:
Isn't this redundant with the level and the presence of the error field?

Reply:
I don't have strong opinions on this. Right now we have some code uses the result field, but we could get the same result by checking `err != ""`.

Reply:
And by "code" i mean Kusto.

Reply:
I'll remove it